core/orders.py

```diff
@@ -149,9 +149,8 @@ def order_to_dict(self):
             }
             if self.order_type in [OrderType.LIMIT, OrderType.STOP_LIMIT]:
                 res.update({"price": self.price})
-                # TODO: restore this property
-                # if self.post_only:
-                #     res.update({"execInst": "ALO"})
+                if self.post_only:
```

Comment:
do we want to reject market orders with this flag?

Reply:
Good point.
Yes, I think we should make `post_only` False as default and raise if it's turned True with order type Market